AndroidAnnotations/androidannotations-core/androidannotations-test/src/main/java/org/androidannotations/test/receiver/ActivityWithReceiver.java

```diff
@@ -58,4 +61,10 @@ protected void onBroadcastWithTwoActions(Intent intent) {
 			action2Fired = true;
 		}
 	}
+
+	@Receiver(actions =  "org.androidannotations.ACTION_1")
```

Comment:
also a string instead of a constant as in ReceiverWithActions. i think you did that to not polute this PR with extracting the other strings?